wasp/boards/simulator/main.py

```diff
@@ -22,4 +22,5 @@
         'artist': 'Dreams of Bamboo',
     })
 
+wasp.system.set_theme(b'\xef{\xef{\xef{\xef{\x00\xf8\xe0\x07<\xe7iJiJ\xef{\xb6\xb5\xb6\xbd\xff\xff')
```

Comment:
This is test code... it should not be commited.

Reply:
this is not quite test code - it's the default theme, which uses the current wasp-os colors

Reply:
removed in next commit